_sections/scientific_days.md

```diff
@@ -10,13 +10,22 @@ with techniques such as those in nilearn. As the goal is to inspire and
 sketch the future of nilearn, the talks are not restricted to studies
 that have been performed using nilearn.
 
-Speaker | Presentation Title
---------|--------------------
-[Carsen Stringer](http://www.gatsby.ucl.ac.uk/~cstringer/)  |  Rastermap: A Visualization Tool for High-dimensional Neural Data
-[Eva Dyer](https://dyerlab.gatech.edu/people/pi-profile/)  | Comparing High-Dimensional Neural Recordings Across Time, Space, and Behavior
-[Jo Etzel](https://sites.wustl.edu/ccplab/people/jo-etzel/) | Pattern Similarity Analyses of Frontoparietal Task Coding: Individual Variation and Genetic Influences
-[Luke J Chang](https://pbs.dartmouth.edu/people/luke-j-chang-0)  | *to be announced*
-[Aki Nikolaidis](https://childmind.org/bio/aki-nikolaidis-phd/)  | Bagging Improves Reproducibility of Functional Parcellation of the Human Brain
-[Sylvia Villeneuve](http://www.villeneuvelab.com/en/home/)/[Alexa Pichet Binette](https://www.pubfacts.com/author/Alexa+Pichet-Binette) | Predicting Functional Brain Aging in Preclinical Alzheimer’s Disease
+ *Schedule is tentative and subject to changes*
 
 
+EDT<sup>&#42;</sup> | CET<sup>&#42;</sup> | Speaker | Presentation Title
+----|-----|---------|-------------------
+09:00 | 15:00 | [Sylvia Villeneuve](http://www.villeneuvelab.com/en/home/)/[Alexa Pichet Binette](https://www.pubfacts.com/author/Alexa+Pichet-Binette) | Predicting Functional Brain Aging in Preclinical Alzheimer’s Disease
+09:30 | 15:30 |  [Carsen Stringer](http://www.gatsby.ucl.ac.uk/~cstringer/) | Rastermap: A Visualization Tool for High-dimensional Neural Data
```

Comment:
It feels like some people have 45 minutes and some have 30mn. That doesn't feel right to me.

Reply:
I'm guessing @thomasbazeille took this from the Software day, where the first talk is a 30min intro to the purpose of the day. Could we add a similar one, here, and shift the talks down ?

Reply:
> I think we might give 40' to everybody (30' talks + 10' questions ?)

+1 on 40' talks

>  the first talk is a 30min intro to the purpose of the day. Could we add a similar one, here, and shift the talks down ?

+1 as well on an introduction to give more momentum to the day (and a bit more flexibility for late people ;) ) 

Who should do it ? @GaelVaroquaux or @bthirion I guess ?
I guess the topic should be along the lines of "Great software shapes great research" wdyt ? 

But I would rather keep the schedule in 3 blocks than having one talk alone, so maybe a short introduction (10 mins) would do it ? 

For now this compromise makes for a short first biobreak (10 mins) but maybe it's ok ?

Reply:
Also as the days finishes early, a longer lunch break would be more comfortable for people and in case we have some delay (which may arise because of questions by visioconf) 

... but a too long break might lose us people going to something else.

Would 1h30 be better ?